### DISCUSSION TITLE
Multiple Instances?

### DISCUSSION THREAD

**Discussion by newoski** [JonathanTreffler/backblaze-personal-wine-container#136] [Q&A]
Has anyone tried running multiple instances of the docker at once, all signed into the same account? Would that be a way to speed up smaller files?

**Comment by traktuner** [comment]
Hey,
unfortunately that's not possible. Every Backblaze installation has its own unique ID. So it would be like backing up 2 different devices.
It also technically can't work because Backblaze puts its own unique volume IDs into the root of every volume (.bzvol folder) so that would also collide.

**Comment by newoski** [comment]
Thanks!
